docs/GettingStarted.md

```diff
@@ -846,6 +847,15 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 | ``tracer`` | A ``Datadog::Tracer`` instance used to instrument the application. Usually you don't need to set that. | ``Datadog.tracer`` |
 | ``databases`` | Hash of tracer settings to use for each database connection. See [ActiveRecord](#activerecord) for more details. | ``{}`` |
 
+Supported versions
```

Comment:
I like the added clarity here. Nice improvement!

Minor thing but should we bold this "Supported versions" text? As a sub-header?

Reply:
Just FYI, I shamelessly stole this version table and the regex version check here #535 :) So when you end up rebasing on that, these changes should be moot.